k8s/cert-manager/crds/kustomization.yaml

```diff
@@ -100,82 +116,27 @@ patches:
+  #"iam.gke.io/gcp-service-account=dns01-solver@$PROJECT_ID.iam.gserviceaccount.com"
   - target:
-      kind: Namespace
+      version: v1
+      kind: ServiceAccount
       name: cert-manager
-    # use ~1 instead of / in path (see https://github.com/kubernetes-sigs/kustomize/issues/1256)
+      namespace: cert-manager
     patch: |-
       - op: add
-        path: /metadata/labels/istio.io~1rev
-        value: asm-managed
-  # Using the minimum pod resources https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-resource-requests#compute-class-min-max
-  - patch: |
-      apiVersion: apps/v1
-      kind: Deployment
-      metadata:
-        name: cert-manager
-      spec:
-        template:
-          spec:
-            containers:
-              - name: cert-manager-controller
-                resources:
-                  requests:
-                    cpu: 200m
-                    memory: 400Mi   
-              - name: istio-proxy
-                image: auto
-                resources:
-                  requests:
-                    cpu: 50m
-                    memory: 112Mi   
-    target:
-      kind: Deployment
-      name: "cert-manager"
-  - patch: |
-      apiVersion: apps/v1
-      kind: Deployment
-      metadata:
-        name: cert-manager-cainjector
-      spec:
-        template:
-          spec:
-            containers:
-              - name: cert-manager-cainjector
-                resources:
-                  requests:
-                    cpu: 200m
-                    memory: 400Mi   
-              - name: istio-proxy
-                image: auto
-                resources:
-                  requests:
-                    cpu: 50m
-                    memory: 112Mi   
-    target:
-      kind: Deployment
-      name: "cert-manager-cainjector"
-  - patch: |
-      apiVersion: apps/v1
-      kind: Deployment
-      metadata:
-        name: cert-manager-webhook
-      spec:
-        template:
-          spec:
-            containers:
-              - name: cert-manager-webhook
-                resources:
-                  requests:
-                    cpu: 200m
-                    memory: 400Mi   
-              - name: istio-proxy
-                image: auto
-                resources:
-                  requests:
-                    cpu: 50m
-                    memory: 112Mi   
-    target:
-      kind: Deployment
-      name: "cert-manager-webhook"
+        path: "/metadata/annotations"
+        value:
+          iam.gke.io/gcp-service-account: dns01-solver@phx-01h4rr1468rj3v5k60b1vserd3.iam.gserviceaccount.com
+
+  # # enable istio injection for the namespace
+  # - target:
+  #     kind: Namespace
+  #     name: cert-manager
+  #   # use ~1 instead of / in path (see https://github.com/kubernetes-sigs/kustomize/issues/1256)
+  #   patch: |-
+  #     - op: add
+  #       path: /metadata/labels/istio.io~1rev
+  #       value: asm-managed
```

Comment:
Do you think there's a chance we might find some reason to re-enable istio injection in `cert-manager`? If not, I'd recommend deleting this block just so someone skimming the file in the future isn't confused.

Reply:
No problem I can delete it, Thanks :)